velox/docs/functions/spark/map.rst

```diff
@@ -6,18 +6,18 @@ Map Functions
 
     Returns value for given ``key``, or ``NULL`` if the key is not contained in the map.
 
-.. spark:function:: map() -> map(unknown, unknown)
+.. spark:function:: map(K0, V0, K1, V1, ...) -> map(K,V)
```

Comment:
This signature is a bit strange. What does map(K0, V0, K1, V1, ...) mean?

Reply:
Does this function take a variadic list of arguments? I don't think Velox supports that unless K and V are the same type.

Reply:
I followed the spark doc. I just realized this should be function signature, then it should be 

`map(K, V, K, V, ...) -> map(K,V)`